lua/hbac/command.lua

```diff
@@ -0,0 +1,41 @@
+local state = require("hbac.state")
+
+local M = {
+	subcommands = {}
+}
+
+M.subcommands.close_unpinned = function()
+	local curbufnr = vim.api.nvim_get_current_buf()
+	local buflist = vim.api.nvim_list_bufs()
+	for _, bufnr in ipairs(buflist) do
+		if vim.bo[bufnr].buflisted and bufnr ~= curbufnr and not state.is_pinned(bufnr) then
+			vim.cmd('bd ' .. tostring(bufnr))
+		end
+	end
+	print("Hbac: Closed unpinned buffers")
+end
+
+
+M.subcommands.toggle_pin = function()
+	local bufnr = vim.api.nvim_get_current_buf()
+	print("Hbac: Buffer " .. bufnr .. " " .. state.toggle_pin(bufnr))
+end
+
+M.vim_cmd_name = "Hbac"
+
+M.vim_cmd_func = function(cmd)
+	if M.subcommands[cmd] then
+		M.subcommands[cmd]()
+	else
+		print("Unknown Hbac command:", cmd)
+	end
+end
```

Comment:
Renamed `args` param to `cmd` to be consistent and less confusing in the `nvim_create_user_command` function. Please confirm this is acceptable before merge.

Reply:
how about `name`? just throwing it out there bc the multiple versions of the word command throughout can be confusing

Reply:
The reason i changed it to `cmd` was to be consistent with the `cmd.args` I introduced above, but since we'll do `args.args` then we can change it back to how it was in your refactor, `arg`.